types/index.d.ts

```diff
@@ -0,0 +1,24 @@
+export = parserWrapper;
+
+interface PostHTMLNode {
+    tag: string;
+    attrs: { [key: string]: string };
+    content: Array<string | PostHTMLNode>
+}
+
+declare function parserWrapper(html: string): PostHTMLNode[];
+
+declare namespace parserWrapper {
+    export var defaultOptions: {
+        lowerCaseTags: boolean;
+        lowerCaseAttributeNames: boolean;
+    };
+
+    export var defaultDirectives: {
+        name: string;
+        start: string;
+        end: string;
+    }[];
+}
+
+
```

Comment:
2 empty line. fix it in `.editorconfig`

Reply:
i create new issue on this and remove 2 empty lines